docs/guides/primitives-rest-api.mdx

```diff
@@ -479,6 +479,141 @@ else:
   </TabItem>
 </Tabs>
 
+<span id="start-sampler-parms"></span>
+## Sampler primitive with REST API and parameterized circuits
+
+### 1. Initialize the account
+
+Because Qiskit Runtime Sampler is a managed service, you first need to initialize your account. You can then select the device you want to use to run your calculations on.
+
+Find details on how to initialize your account, view available backends, and invalidate tokens in this [topic](./setup-channel#set-up-to-use-ibm-quantum-platform-with-rest-api).
+
+### 2. Define parameters
+
+```python
+import requests
+import qiskit_ibm_runtime
+# Imports from Qiskit Runtime
+from qiskit_ibm_runtime import QiskitRuntimeService
+from qiskit.transpiler.preset_passmanagers import generate_preset_pass_manager
+
+from qiskit.qasm3 import dumps
+from qiskit import QuantumCircuit
+from qiskit.circuit import Parameter
+from qiskit import transpile
+
+service = QiskitRuntimeService(channel='ibm_quantum')
```

Comment:
Is `channel` necessary? We don't normally set this, do we?

Reply:
It's not really necessary, but we specify it in several examples.  If you have access to cloud and IQP, it defaults to Cloud, which can be a surprise.  Since it was in the code I was given, I'll probably just leave it.